### PR TITLE
Adhoc performance study

### PR DESCRIPTION
**Pull request by antony-gigapixel-imaging**
Jira: BIOINF-1426

### CODE REVIEW

File: roles/pleno_droid/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    virtualenv: "{{ pleno_droid_droid2_venv }}"
+    virtualenv_site_packages: true
+    virtualenv_command: /opt/miniconda3/bin/python3.11 -m venv
+
+- name: (Droid2) Install droid and dependencies
+  ansible.builtin.pip:
+    name: "droid==v{{ pleno_droid_droid2_version }}"
+    state: present
+    virtualenv: "{{ pleno_droid_droid2_venv }}"
+    extra_args: "--extra-index-url https://pypi.plenoinc.com/pypi/simple/"
```
Comment (antony-gigapixel-imaging) [inline roles/pleno_droid/tasks/main.yml:21]: Note to self: 

```
ERROR: Could not find a version that satisfies the requirement droid==v2.0.1 (from versions: 1.4.21, 1.4.22)
```

Comment (antony-gigapixel-imaging) [reply roles/pleno_droid/tasks/main.yml:21]: Resolved by installing `pleno_droid` instead of `droid`. closing.

**Comment by bo-at-pleno** [comment]
https://github.com/Pleno-Inc/pleno-droid/releases/tag/v2.0.1

we don't publish to public pypi you need to do `--extra-index-url https://pypi.plenoinc.com` and it's also password protected, see this page for details https://plenoinc.atlassian.net/wiki/spaces/SI/pages/212008963/Pleno+Pypi+Repo+hosts+internal+pip+packages

**Review by bo-at-pleno — APPROVED** [review]
This is cool, i'll tag @dylanpleno  and @panfengli-work  here to learn this technology so they can use it too in the future for RASS or other bridging study related work.

**Pull request merged**